.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+# Description
+
+<!-- Please write a summary of your changes and why you made them.-->
+
+## Links to any relevant issues
+
+<!-- Reference any related issues.-->
+
+## Type of change
+
+<!-- Choose a type of change, and delete any options that are not relevant.-->
+
+- [ ] Bug Fix
+- [ ] Enhancement
+- [ ] Feature
+- [ ] Refactor
+- [ ] Documentation Fix
+- [ ] Other:
```

Comment:
```suggestion
```

Is this needed? We don't categorize our changelog entries today, thus I am not sure this would be needed.